infrastructure/cluster/flux-v2/eos/eosfuse-daemonset.yaml

```diff
@@ -0,0 +1,53 @@
+# DaemonSet with the latests CERN EOS client.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: eosfuse
+  namespace: jhub
+spec:
+  selector:
+    matchLabels:
+      app: eosfuse
+  template:
+    metadata:
+      labels:
+        app: eosfuse
+    spec:
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      hostPID: true
+      hostIPC: true
+      tolerations:
+        - key: jupyter-role
+          operator: Equal
+          value: singleuser
+          effect: NoSchedule      
+      nodeSelector:
+        # makes sure EOS eulake fuse mount is done on nodes designated to have the fuse mount
+        jupyter-role: singleuser
+      containers:
+        - name: eos-client
+          image: gitlab-registry.cern.ch/linuxsupport/cc7-base:latest
+          command: ["sleep", "100000"]
+          # securityContext:  # To be cleaned if eos mounted correctly 
+          #   seLinuxOptions:
+          #     type: "spc_t"
+          #   privileged: true
+          #   capabilities:
+          #     add:
+          #       - SYS_ADMIN
+          #       - NET_ADMIN
+          volumeMounts:
+            - name: eos-eulake
+              mountPath: /eos
+              mountPropagation: HostToContainer
+      volumes:
+        - name: eos-eulake
+          # Note the particularity of the host path being /var/eos, not /eos.
+          # The path inside the container will be /eos with this sample setup.
+          hostPath:
+            path: /var/eos-eulake-home-test/
+      securityContext:
```

Comment:
I think this is in the wrong place now, you commented it out above?

Reply:
There are two places where the `securityContext` is applied. Within the specs; in the [eos team manifest](https://gitlab.cern.ch/kubernetes/automation/charts/cern/raw/master/eosxd/examples/eos-client-example.yaml) or inside de `containers`, in the [old escape cluster](https://gitlab.cern.ch/escape-wp2/flux-rucio/-/blob/master/escape/jupyterhub/daemonset/eosfuse.yaml#L30).

Trying how the eos team way, as you suggested too